fixed register page UI issue #43

sherlock6147 opened the pull request:
Fixed issue  https://github.com/lugnitdgp/ReviewBook/issues/43

sherlock6147 committed:
fixed register page UI

kumanik commented:
Send screenshots of the UI changes you did
Also you didn't change the login forms try to create a uniform UI

sherlock6147 committed:
fixed UI for login page

sherlock6147 commented:
Moved the css for register.html from base_css.html to register.html for dry code and updated the UI for the login page and changed the nav button color for better UI

register.html
![Screenshot from 2021-01-26 15-24-18](https://user-images.githubusercontent.com/57949015/105832180-a2e90480-5fed-11eb-9166-fb0c9297c698.png)

login.html
![Screenshot from 2021-01-26 15-35-52](https://user-images.githubusercontent.com/57949015/105832274-c14f0000-5fed-11eb-8a33-72477375e1f3.png)

kumanik commented:
Make the register form centered and labels in one row and field in another

sherlock6147 committed:
styled the register form and made it responsive

sherlock6147 commented:
i added some more styles to the registration form and removed the register button from register.html . This is how it looks after the changes
![Screenshot from 2021-01-27 23-59-37](https://user-images.githubusercontent.com/57949015/106036760-bd110880-60fb-11eb-95fd-4e1ce8ff677c.png)
I also added an already registered button which redirects to login.html
